### PR TITLE
chore: Create files to improve developer experience and keep consistency

### LINKED ISSUE #13: chore: Criar definição de arquivos para formatação automática de código
Descrição da ideia de forma simples 

## Objetivo
Qual o objetivo da ideia o que ela entrega de valor

- Conseguirmos manter o mesmo styleguide de formatação de código entre os devs que contribuirem com o projecto.

### Arquivos sugeridos

- eslint
- editorconfig
- prettier

### PR DESCRIPTION
**Pull request by samuelsilvadev**
## Goal

- Improve our developer experience and more than that keep consistency between all environments.

## Note

- As this PR changes a lot of of files and a little about the logic, it's very important that we review it with care.

## Changes

- Created `.editorconfig`, `.eslintrc`.
- Added new dependencies, eslint, prettier.
- Fixed project vunerability.
- Fixed dependencies and devDependencies in package.json.

##  Related issue

close #13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,52 +1,62 @@
...
-    "style-loader": "^0.23.1",
-    "webpack": "^4.29.3",
-    "webpack-cli": "^3.2.3",
-    "webpack-dev-server": "^3.1.14"
-  },
-  "dependencies": {
-    "react-datepicker": "^2.4.0",
-    "tar": ">=4.4.2"
-  }
+	"name": "webpack-react-tutorial",
```
Comment (cassunde) [inline package.json]: @samuelsilvadev you can change this name, maybe 'taok-front'

File: src/js/components/BarraTotal/index.js [hunk 2]
```diff
@@ -1,107 +1,122 @@
...
+                this.setState({
+                    labels: res.data[0].labels,
+                    valores: res.data[0].valores,
+                });
+            });
     }
 
-    calculaAcumulado(valores){
+    calculaAcumulado(valores) {
+        // const total = 0.0;
```
Comment (cassunde) [inline src/js/components/BarraTotal/index.js]: @samuelsilvadev you can remove this comments

Comment (samuelsilvadev) [reply src/js/components/BarraTotal/index.js]: sure, nice catch!!

File: package.json (hunk 1 above)
Comment (samuelsilvadev) [reply package.json]: perfect!!

**Comment by samuelsilvadev** [comment]
@cassunde I already updated the code, when you can take a look :D

**Pull request merged**